db-cleaner/cfn.yaml

```diff
@@ -87,7 +87,7 @@ Resources:
       Handler: Provided
       MemorySize: 128
       Role: !GetAtt ExecutionRole.Arn
-      Runtime: java8
+      Runtime: provided
```

Comment:
This is unrelated but now we can finally cloudform a lambda with a provided runtime. That avoids having to manually change the value in the console after applying the value (this is the rust lambda)